crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -58,6 +58,7 @@ spec:
           spec:
             type: object
             required:
+            - priority
```

Comment:
This is not a required field since we are trying to make it backwards compatible. Catalog sources without priority specified will be assigned a value.